lib/linux_admin/hosts.rb

```diff
@@ -26,6 +26,10 @@ def add_alias(address, hostname, comment = nil)
 
     alias_method :update_entry, :add_alias
 
+    def set_loopback_hostname(hostname, comment = nil)
+      ["::1", "127.0.0.1"].each { |address| add_name(address, hostname, true, comment, false) }
```

Comment:
Should we worry about this method adding duplicates if the hostname already exists for any of the loopback addresses?